packages/concerto-core/api.txt

```diff
@@ -55,17 +55,22 @@ class Concerto {
    + string getNamespace(obj) 
 }
    + object setCurrentTime() 
+class DcsIndexWrapper {
```

Comment:
Do we want this to public?

Reply:
After reviewing the rest of the code, I think not. This is an internal helper class.

Reply:
Moved to `DecoratorManager.js`